docs/conf.py

```diff
@@ -161,16 +161,16 @@
     "dudir": ("http://docutils.sourceforge.net/docs/ref/rst/directives.html#%s", ""),
 }
 
+object_description_options_map = {}
+object_description_options_patterns = []
+
 # BEGIN: sphinx_immaterial.format_signatures extension options
-clang_format_signatures_domain_styles = {
-    "cpp": """{
-    BasedOnStyle: LLVM,
-    ColumnLimit: 68,
-    }""",
-}
+object_description_options_patterns.append(("cpp:.*", {"BasedOnStyle": "LLVM"}))
 # END: sphinx_immaterial.format_signatures extension options
 
-html_wrap_signatures_with_css = ["py"]
+object_description_options_patterns.append(
+    ("py:.*", dict(wrap_signatures_with_css=True))
+)
```

Comment:
I just pushed the v0.5.0 tag without knowing your work on this was changing the config option names. These config option changes seem like a breaking change, so I'd use a major version bump for the next tag.

Reply:
It is true that it is a breaking change, but on the other hand if we get this in soon I doubt too many people will have used the new options yet.

Also v0 generally means no stability guarantees.

But yeah it could make sense to increase to v1.

Reply:
I'm ok with bumping minor version because I'd also consider v0 as "beta" state.